### PR TITLE
More ginkgo migration

### PR DESCRIPTION
**Pull request by baude**
* attach
* run_exit
* save
* tag
* version
* run_privileged -> privileged

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
LGTM once tests green

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@rh-atomic-bot retry

**Comment by mheon** [comment]
@baude Looks like the "sh -c exit..." test is having issues

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
bot, retest this please

File: test/e2e/run_ns_test.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		podmanTest.RestoreArtifact(fedoraMinimal)
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+
+	})
+
+	It("podman run pidns test", func() {
+		session := podmanTest.Podman([]string{"run", fedoraMinimal, "bash", "-c", "echo $$"})
```
Comment (TomSweeneyRedHat) [inline test/e2e/run_ns_test.go]: These used to run on Alpine, should we still have a few here?  Or perhaps another set that run on Alpine too?  Just from an overall test perspective, how many tests should we run on alpine, centos, fedora (maybe rhel) containers?  I don't have an answer nor need one, just throwing it up for consumption.

**Comment by TomSweeneyRedHat** [comment]
@baude unhappy tests fwiw.

**Comment by mheon** [comment]
@baude Travis says you have a build error in your tests?

**Comment by baude** [comment]
bot, retest this pleaase

File: test/e2e/run_ns_test.go (hunk 1 above)
Comment (baude) [reply test/e2e/run_ns_test.go]: the tests werent actually being run, so they never worked ... when I tested them in ginkgo and locally, alpine didn't cut it.

Comment (baude) [reply test/e2e/run_ns_test.go]: also, i worked it to cache fedora-minimal and only unpack it when needed so overall we speed up

Comment (TomSweeneyRedHat) [reply test/e2e/run_ns_test.go]: good to know, thanks for the 411.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
Homu is hosed, Merging manually.

**Pull request merged**